api/common.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package containerd.v1;
+
+import "google/protobuf/empty.proto";
+
+message Process {
```

Comment:
Rather than having this called the `common` package, which usually turn into a mess, let's call this the process package.